cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -136,6 +141,10 @@ runQueryCmd cmd =
     QueryUTxO' consensusModeParams qFilter networkId mOutFile ->
       runQueryUTxO consensusModeParams qFilter networkId mOutFile
 
+mapQueryError :: QueryError -> ShelleyQueryCmdError
```

Comment:
These should be additional data constructors in `ShelleyQueryCmdError`. Generally we use `firstExceptT ...` to wrap the underlying error types.